### ISSUE TITLE
DayPicker Navigation button problem in RTL

### ISSUE THREAD

**Issue by man4toman** [WordPress/gutenberg#27535]
DayPicker Navigation button has a problem in RTL, in WordPress 5.5.3, please see the attachment image.

![image](https://user-images.githubusercontent.com/935397/101285300-a801b280-37f9-11eb-9855-f47c05236f3b.png)

The problem came from:
```
.components-datetime__date .DayPickerNavigation_leftButton__horizontalDefault {
    right: 0;
}
```

It must changes to:
```
.components-datetime__date .DayPickerNavigation_leftButton__horizontalDefault {
    right: auto;
}
```

After this change:
![image](https://user-images.githubusercontent.com/935397/101285390-28281800-37fa-11eb-8e04-5b0f7b699a13.png)

**Comment by gaavar** [comment]
Hi! Can I work on this issue?

**Comment by gaavar** [comment]
@talldan I am interested in fixing this bug! I located the appropriate code that requires revision. However, to visually test the change, I am supposed to render the DateTime component. I attempted to follow the starter-block tutorial (`npx @wordpress/create-block starter-block`) and use something like

`export default function Edit( { className } ) {
	return (
		<p className={ className }>
			{ __( 'Starter Block – hello from the editor!', 'starter-block' ) }
			<DateTimePicker
			currentDate={ date }
			onChange={ ( date ) => setState( { date } ) }
			is12Hour={ is12HourTime }
			/>
		</p>
	);
}`

in the edit.js.

What is a simple way to render the DateTime?

**Comment by gaavar** [comment]
I installed an rtl tester plugin to test the idea of setting left button's css to `right: auto;`. However, the settings result in:

![image](https://user-images.githubusercontent.com/33789105/102038677-e563d280-3ded-11eb-8cc8-1767fa2db0b2.png)

I tried a bunch of values. Not sure how to fix this.

**Comment by gaavar** [comment]
@talldan I was able to align the calendar buttons correctly in RTL (tested using an RTL plugin)
![image](https://user-images.githubusercontent.com/33789105/102039706-a5521f00-3df0-11eb-822b-dc5e312fa208.png)


However, in the LTR version, I obtain this:
![image](https://user-images.githubusercontent.com/33789105/102039671-8d7a9b00-3df0-11eb-939b-0106f8d11826.png)

I am not sure how the CSS can be isolated to triggered differently for RTL/LTR

**Comment by noisysocks** [comment]
@gaavar: There's a few approaches.

You can check for the presence of the `.rtl` class on `<body>` which is added by WordPress when the current language is a RTL language.

Alternatively, an even better idea is to use a rtlcss directive. 

```css
.components-datetime__date .DayPickerNavigation_leftButton__horizontalDefault {
    right: 0 /*rtl:auto*/;
}
```

https://rtlcss.com

**Comment by hsingyuc** [comment]
Hey guys, just a heads up that I'm working on this one here https://github.com/WordPress/gutenberg/pull/27730 alongside https://github.com/WordPress/gutenberg/issues/27536.

**Comment by brentswisher** [comment]
Working through the [5.7 Must Haves](https://github.com/WordPress/gutenberg/projects/52), and it looks like this was fixed in #27730 @jasmussen or @hsingyuc can you confirm this issue can also be closed?

**Comment by jasmussen** [comment]
Yep, good one!
